ripple.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
 
   gem.add_dependency "riak-client", "~> 2.6"
-  gem.add_dependency "activesupport", "~> 5.0.7"
-  gem.add_dependency "activemodel", "~> 5.0.7"
+  gem.add_dependency "activesupport", "<= 5.1.7"
```

Comment:
Why the constraint operator switch here?

Reply:
So that we can support older versions of rails. We should probably do >= 5.0.x also.

Reply:
actually...after doing some more testing, the changes required for callback propagation won't work in versions before 5.1 so either we need to 
1) add in logic to support both in the `propagate_callbacks_to_embedded_associations` method, or
2) lock to a 5.1.x version

for now, I switched the operator back, but if we think supporting both is a better option, I can update that